docs/transactions.md

```diff
@@ -128,6 +128,7 @@ Adds a record about a Vendor.
   - companyLegalName: `string` -  Legal name of the vendor company
   - companyPreferredName: `optional(string)` -  Preferred name of the vendor company
   - vendorLandingPageURL: `optional(string)` -  URL of the vendor's landing page
```

Comment:
Please make sure that we have schemaVersion in docs for the following commands:
- ADD_PKI_REVOCATION_DISTRIBUTION_POINT
- UPDATE_PKI_REVOCATION_DISTRIBUTION_POINT
- Approve/Reject/Revoke wrappers for PKI commands (see https://github.com/zigbee-alliance/distributed-compliance-ledger/pull/552#discussion_r1533681328)